packages/sync/README.md

```diff
@@ -17,3 +17,140 @@ To enable debug please execute in console
+conflictStrategy | The conflict resolution strategy your client should use | N/A
+customLinkBuilder | Enables providing custom Apollo Link for processing requests | See `LinksBuilder`
+networkStatus | Implementation of `NetworkStatus` Interface | See `WebNetworkStatus` and `CordovaNetworkStatus`
+mutationsQueueName | The name to store requests under in your offline queue | "offline-mutation-store"
+squashing | Whether or not you wish to squash mutations in your queue | true
+
+
+# Creating a Client
+```javascript
+let client = createClient(config);
+```
+
+
+## Conflict strategies
+You can provide your custom conflict resolution strategies to the client in the config by using the provided `ConflictResolutionStrategy` type from the SDK. An example custom strategy is shown below.
+
+```javascript
+let clientWins = (serverData, clientData) => {
+  return Object.assign(server, client);
+};
+```
+
+To use this strategy pass this function as conflictStrategy in your config object like so:
+
+```javascript
+let config = {
+...
+  conflictStrategy: clientWins
+...
+}
+```
+
+## Implementing Custom Apollo Links
+To use your own custom apollo links to create your own network flow, simply follow the documentation for creating links here: https://www.apollographql.com/docs/link/index.html. You can pass this building mechanism to your client in the config, under the `customLinkBuilder` parameter.
+
+```javascript
+export const linkBuilder: LinkChainBuilder = (): ApolloLink => {
+    const httpLink = new HttpLink({ uri: "someUri" });
+    const customLink = new YourCustomLink();
+
+    let links: ApolloLink[] = [customLink, httpLink];
+
+    let compositeLink = ApolloLink.from(links);
+    return compositeLink;
+  };
+```
+
+## Implementing Custom Network Status checks
+To use your own custom netork checks, implement the NetworkStatus interface which provides two functions;
+
+```javascript
+  onStatusChangeListener(callback: NetworkStatusChangeCallback): void;
+
+  isOffline(): boolean;
+```
+
+## Offline Mutations
+
+AeroGear Sync SDK provides offline mutations out of the box. By default it uses localStorage to do this and mutations are stored under the cache key of 'offline-mutations-queue'.
+
+### Online Only Queries
+To ensure certain queries are not queued and are always delivered to the network layer, you must make use of Graphql directives. To do so define a directive on your schema type definitions like so:
+
+```
+directive @onlineOnly on FIELD
+```
+
+Next, on your client, ensure the query has the annotation attached like so:
+
+```
+exampleQuery(...) @onlineOnly {
+  ...
+}
+```
+
+### Squashing Queries
+By default AeroGear Sync SDK provides squashing of mutations out of the box. This means that if your client is offline and the same mutation is run twice with different parameters they will be squashed into one mutation. This is beneficial as the client will not have to queue a large amount of mutations to replay once it returns online.
```

Comment:
Squashing is kinda internal language. I'm reading this as newbie now and first sentence is kinda cryptic. 
We could say:

Changes that are performed on the same object id and mutation name will be joined together when offline. 
Process is called squashing or something like this